mlflow/models/evaluation/default_evaluator.py

```diff
@@ -565,30 +566,33 @@ def _is_string(value):
     return isinstance(value, str)
 
 
-def _evaluate_extra_metric(extra_metric_tuple, eval_fn_args):
+def _evaluate_metric(metric_tuple, eval_fn_args):
     """
-    This function calls the `extra_metric` function and performs validations on the returned
+    This function calls the `metric` function and performs validations on the returned
```

Comment:
Nit: There's no variable called `metric`. If there were, we should've used double backticks

```suggestion
    This function calls the metric function and performs validations on the returned
```